single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -499,9 +499,19 @@ protected void setMetadata(GraphMetadata metadata, Node node, String nodeId, Vol
             }
         }
 
+        // to generate the real component type in node metadata for 2WT and 3WT transformers
+        // (actually, the component type is set to 'LINE' for 3WT feeder leg nodes and for 2WT feeder leg nodes in
+        //  substation diagram, for internal reasons)
```

Comment:
I checked it, and unless I forgot something there's no internal reason (anymore?) to set the component type to `LINE` for the leg nodes.